backend/curriculum_tracking/models.py

```diff
@@ -1619,17 +1619,46 @@ def get_users_that_reviewed_open_prs(self):
 
         return [review.user for review in reviews]
 
-    def request_user_can_start(self):
-        """can the user start this card?
+    def request_user_can_start(self, user=None):
+        """
+        Check if current user can start this card
+
+        This function is only used in template rendering and we are using threadlocals,
+        that is why we need to avoid argument parameters unless they're for testing purposes
+        """
+        if user is None:
+            from threadlocal_middleware import get_current_user
+
+            user = get_current_user()
+
+        if user is not None:
+            request_user_is_assignee = self.assignees.first() == user
+
+            if request_user_is_assignee and self.can_start():
```

Comment:
can_start() takes in force=Bool, for when user when super rights, do we not need that here as well?

Reply:
I've accounted for cards that can be "force started" and allowed super users and team managers to force start cards that can be "force started."